### PR TITLE
Mobile portfolio

### PR DESCRIPTION
**Pull request by KayLemba**
In this project  I created a mobile portfolio using HTML and Cascading style sheet (CSS), and I did the following;

1. Update index.html file.
2. Update styles.css file.
3. Modify README.md file.
4. Add new screenshot .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (SajjadAhmad14) [inline README.md:2]: Please add all unnecessary files and folders like `DS_store` to `.gitignore`.

File: index.html [hunk 2]
```diff
@@ -32,12 +28,154 @@ <h1 class="about-me">
...
           <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
           <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
         </ul>
       </div>
-    </section>
\ No newline at end of file
+    </section>
+    <section class="work-section">
+      <div class="project-card">
```
Comment (SajjadAhmad14) [inline index.html:41]: Please align the project details section with the image horizontally as it's in the design.
Yours:
details
Orignal:
orignal

File: index.html [hunk 3]
```diff
@@ -32,12 +28,154 @@ <h1 class="about-me">
 
         <p class="lets-connect"> LET'S CONNECT</p>
```
Comment (SajjadAhmad14) [inline index.html:29]: Please use `hx` tag for the `Let's connect` heading.

File: README.md (hunk 1 above)
Comment (KayLemba) [reply README.md:2]: Hello Im not quite clear on this  please help

Comment (KayLemba) [reply README.md]: done

Comment (KayLemba) [reply README.md]: Thank You

File: index.html (hunk 2 above)
Comment (KayLemba) [reply index.html:41]: Thanks for the correction @SajjadAhmad14

**Pull request merged**